pymbolic/geometric_algebra/__init__.py

```diff
@@ -388,25 +425,23 @@ def orthogonal_blade_product_weight(a_bits, b_bits, space):
 
 # {{{ multivector
 
-def _cast_or_ni(obj, space):
+def _cast_or_ni(obj: Any, space: Space) -> MultiVector:
```

Comment:
What's `or_ni` stand for? :confounded:

Reply:
It was *supposed* to mean "not implemented". If you look, there is (was) a bunch of code handling this function returning `NotImplemented`. But I don't think it ever did, so I've killed that code (and renamed this to `_cast_to_mv`).